Url.cpp

```diff
@@ -141,100 +148,98 @@ std::string Url::encode(const std::string& str)
     return encoded_str;
 }
 
-
 void Url::regex_parse()
 {
     std::smatch match;
     if (std::regex_match(url_string, match, URL_REGEX))
     {
-        scheme = match[2].str() + "://";
-        hostname = match[4];
-        path = match[5];
+        scheme        = match[2].str() + "://";
```

Comment:
Can we keep this how it was? Seems odd to do this for normal variables - I would understand for something like a struct or similar